README.rst

```diff
@@ -79,6 +81,35 @@ printed instead.
                             Config file with options to customize how content
                             is generated.
 
+Python Library
+~~~~~~~~~~~~~~
+
+You can also generate STIX objects in a Python script.
+
+You can create single objects of a specified typwusing the generate function of
+an object generator:
+
+.. code-block:: python
+
+    object_generator = stix2generator.create_object_generator()
+    indicator = object_generator.generate("indicator")
+
+You can also use the language_processor object in a similar fashion as the
+command-line tool:
+
+.. code-block:: python
+
+    language_processor = stix2generator.create_default_language_processor()
+    indicator = language_processor.build_graph("Indicator.")
+
+A given configuration file can produce more specific results, if necessary:
+
+.. code-block:: python
+
+    config = stix2generator.generation.Config(optional_property_probability=.25, minimize_ref_properties=False)
```

Comment:
This should use class `stix2generator.generation.object_generator.Config`.  The different generators have their own specific Config classes, which define generator-specific config settings.